test/integration/Session.test.js

```diff
@@ -40,6 +41,11 @@ describe('Session', () => {
                 password: 'tester2',
             },
         })
+        clientSessionToken = createClient({
+            auth: {
+                sessionToken: 'session-token',
+            },
+        })
         clientNone = createClient({
             auth: {},
```

Comment:
Found a bug. `new StreamrClient()` currently fails because `options.auth` is not defined. To replicate the problem just delete this line.